apps/condo/domains/resident/utils/accessSchema.js

```diff
@@ -0,0 +1,72 @@
+const { get, set } = require('lodash')
+
+const conf = require('@open-condo/config')
+const { getRedisClient } = require('@open-condo/keystone/redis')
+const { find } = require('@open-condo/keystone/schema')
+
+const _redisClient = getRedisClient('default', 'cache')
+const DEFAULT_CACHE_TTL_IN_MS = 60 * 60 * 1000 // 1 hour
```

Comment:
We don't migrate resident, so it has no problem with increasing TTL, like in organisations. But I guess average mobile session is under 1 hour, so it's oke